src/module.ts

```diff
@@ -348,6 +348,17 @@ export default defineNuxtModule<ModuleOptions>({
       global: true
     })
 
+    // Use single components chunk
+    nuxt.hook('vite:extendConfig', (config, { isServer }) => {
+      if (Array.isArray(config.build.rollupOptions.output) || isServer) { return }
+      const moduleComponentsDir = resolve('./runtime/components')
+      config.build.rollupOptions.output.manualChunks = (id) => {
```

Comment:
Would you please add a condition to not reassign `manualChunks` if already set? By implementing https://github.com/nuxt/framework/issues/7240 it will cause conflicting behavior.